README.md

```diff
@@ -1 +1,199 @@
-# react-native-sdk-button
\ No newline at end of file
+
+# Yoti Button SDK for React Native
+
+Yoti button component for React Native [Android]([https://github.com/getyoti/android-sdk-button](https://github.com/getyoti/android-sdk-button)) and [iOS]([https://github.com/getyoti/ios-sdk-button](https://github.com/getyoti/ios-sdk-button)).
+
+![Yoti Logo](https://www.yoti.com/wp-content/themes/yoti/assets/images/logo.svg)
```

Comment:
Was this logo intended to be this big? 😃 
![image](https://user-images.githubusercontent.com/1381991/77162916-f2dd6d80-6aa4-11ea-96b5-a2a93eb3e6d1.png)

Reply:
Actually the issue is not with the size, it is with the image itself.
I have just changed it for the one we have in the other public repos. 👍